poly-synth/src/poly-synth.js

```diff
@@ -103,15 +115,47 @@ class PolySynth {
     let reverbInputNode = new GainNode(this.context_);
     let reverbOutputNode = this.createReverbGraph_(reverbInputNode);
 
-    // The client is responsible for connecting |this.output| to a destination.
-    this.output = new GainNode(this.context_);
-
-    this.voiceOutput_.connect(bitcrusherInputNode);
+    this.voiceOutput_.connect(noisegateInputNode);
+    noisegateOutputNode.connect(bitcrusherInputNode);
```

Comment:
My reasoning was that it doesn't really matter since the noise gate isn't going to construct an envelope the synthesizer data anyway. I put the noise gate first since, in theory one could optimize the bit crusher so that it didn't do any work when its input was all 0's.